Common/TDSQASystemAPI/TDSQASystemAPI.csproj

```diff
@@ -114,12 +114,24 @@
     <Reference Include="System.EnterpriseServices">
       <RequiredTargetFramework>4.5</RequiredTargetFramework>
     </Reference>
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Net.Http.Formatting, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="System.Net" />
+    <Reference Include="System.Net.Http">
+      <HintPath>..\..\TISServices\TISServices\packages\System.Net.Http.2.0.20126.16343\lib\net40\System.Net.Http.dll</HintPath>

```

Comment:
Added missing dependencies that were not previously captured by NuGet (which is a package manager program for .NET).